app/src/main/java/se/studieresan/studs/LauncherActivity.java

```diff
@@ -0,0 +1,56 @@
+package se.studieresan.studs;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.PersistableBundle;
+
+import java.util.ArrayList;
+
+import androidx.annotation.VisibleForTesting;
+import androidx.appcompat.app.AppCompatActivity;
+import se.studieresan.studs.data.StudsPreferences;
+import se.studieresan.studs.login.views.LoginActivity;
+
+abstract interface ILauncherActivity {
```

Comment:
I feel like we need another interface. We must prefix them with 'I' for ultimate readability.